common/include/pcl/PCLPointCloud2.h

```diff
@@ -49,6 +49,62 @@ namespace pcl
   public:
     using Ptr = boost::shared_ptr< ::pcl::PCLPointCloud2>;
     using ConstPtr = boost::shared_ptr<const ::pcl::PCLPointCloud2>;
+
+    //////////////////////////////////////////////////////////////////////////
+    /** \brief Removes fields named "_"
+      * \return true if any fields were removed, false otherwise
+      */
+    bool
+    squeeze ();
+
+    /** \brief Inplace concatenate two pcl::PCLPointCloud2
+      * 
+      * This command doesn't remove any fields named "_" (aka marked as skip)
+      * IFF the layout of all the fields in both the clouds is the same. However, if
+      * the order of skip fields is different, relevant fields are selctively copied.
```

Comment:
```suggestion
      * the order and/or the number of skip fields is different, relevant fields are selctively copied.
```

Reply:
Cool. I added a few other modifications apart from this.